docs/references/events.mdx

```diff
@@ -562,3 +562,82 @@ This [Event](/workflows#event) type indicates that a [Workflow Execution](/workf
+| service                          | Service name.                                                                                                                                                                                                                                                                                             |
+| operation                        | Operation name.                                                                                                                                                                                                                                                                                           |
+| input                            | Input for the operation. The server converts this into Nexus request content and the appropriate content headers internally when sending the StartOperation request. On the handler side, if it is also backed by Temporal, the content is transformed back to the original Payload stored in this event. |
+| schedule_to_close_timeout        | Schedule-to-close timeout for this operation. Indicates how long the caller is willing to wait for operation completion. Calls are retried internally by the server.                                                                                                                                      |
+| nexus_header                     | Header to attach to the Nexus request. Note these headers are not the same as Temporal headers on internal activities and child Workflows, these are transmitted to Nexus operations that may be external and are not traditional payloads.                                                               |
+| workflow_task_completed_event_id | The `WORKFLOW_TASK_COMPLETED` event that the corresponding ScheduleNexusOperation command was reported with.                                                                                                                                                                                              |
+| request_id                       | A unique ID generated by the History Service upon creation of this event. The ID will be transmitted with all Nexus StartOperation requests and is used as an idempotency key.                                                                                                                            |
+| endpoint_id                      | Endpoint ID as resolved in the endpoint registry at the time this event was generated. This is stored on the event and used internally by the server in case the endpoint is renamed from the time the event was originally scheduled.                                                                    |
+
+### NexusOperationStarted
+
+This Event type indicates that an Nexus Operation Execution was started.
+The handler’s SDK Worker picked up the Nexus Operation and started processing the Nexus Operation invocation.
```

Comment:
`NexusOperationStarted` only happens for Async Nexus Operations.

As above, there may be multiple types of handler in the future, so would either remove this sentence, or frame this from the context of the caller workflow event history.

```suggestion
This event is added to the caller's workflow history for Asynchronous Nexus Operations, for example one that is picked up by a handler's SDK client and started by a Nexus handler.
This event is not added to the caller's Workflow history for Synchronous Nexus Operations, since they transition directly to [NexusOperationCompleted](#nexusoperationcompleted) or another final state such as [NexusOperationFailed](#nexusoperationfailed) when the response is provided synchronously by the Nexus handler.
```